spec/abstractops.html

```diff
@@ -950,15 +957,12 @@ <h1>ISO 8601 grammar</h1>
           TimeZoneIANANameComponent `/` TimeZoneIANANameTail
 
       TimeZoneIANAName :
```

Comment:
Hmm, it seems misleading to use _TimeZoneIANAName_ for describing a nonterminal that can also match _TimeZoneUTCOffsetName_ (which are **not** IANA time zone names). Can this be collapsed into _TemporalTimeZoneIdentifier_ (whose _TimeZoneNumericUTCOffset_ is just a duplication of _TimeZoneUTCOffsetName_ anyway)?

Reply:
It looks like according to this grammar, now a string like `2022-02-21T09:25-08:00[-08:00]` would be disallowed, because _TimeZoneBracketedAnnotation_ only surrounds _TimeZoneIANAName_ whereas it used to surround _TimeZoneBracketedName_. Can you check this?

Reply:
Was this still the case?

Reply:
it seems that that was already invalid? I tried to parse that using the previous version of the grammar but it seems that it doesn't work. Don't we need a name of some kind ("Etc/GMT+1" or "America/Kentucky/Louisville") instead of a numeric offset like "-08:00"?

Reply:
I disagree that it was already invalid (TimeZoneUTCOffsetName was part of TimeZoneBracketedName) but I think this is now fixed in the current version that you have.

Richard's comment about collapsing TimeZoneIANAName into TemporalTimeZoneIdentifier should still be addressed.

Reply:
done

Reply:
I don't think this is completely done yet, now TimeZoneIANAName duplicates TemporalTimeZoneIdentifier, I'd recommend either removing TimeZoneIANAName, or just making it equal to TimeZoneIANANameTail as it was before.

Additionally as Richard mentioned, the two alternatives inside TemporalTimeZoneIdentifier of TimeZoneUTCOffsetName and TimeZoneNumericUTCOffset expand to the same things.

Finally, if you remove TimeZoneIANAName, you'll have to search for any references to it in the rest of the spec text and replace those with TemporalTimeZoneIdentifier.

Reply:
I've restored `TimeZoneIANAName`. It wasn't my intention to do such an invasive change. How do things look now?

Reply:
Better. I think this is good enough to fix #1993. There is still something wrong with this grammar, but we have #1805 open for that, so I would be OK to consider it out of scope, especially since the slides I've prepared for plenary only mention the problem in #1993. @gibson042 are you OK with that?